ksql-rest-app/src/main/java/io/confluent/ksql/rest/entity/QueryDescription.java

```diff
@@ -34,7 +33,7 @@
 
 public class QueryDescription {
 
-  private final QueryId id;
+  private final String id;
```

Comment:
Why do we need to change this to a `String`? `QueryId` describes what it is, `String` doesn't. I presume there is a valid reason?

Reply:
all queryid contains is the id. it makes dealing with this field annoying

```    "queryDescriptions": [
      {
        "id": {
          "id": "CSAS_BLAH"
        },
```

Reply:
I think rendering QueryId as an API object is also bad because its not part of the entities package, which makes the api more prone to breakage (since someone may change QueryId without realizing its used to render api objects).

Reply:
You can add a _thing_ to Jackson so that you can leave it as a `QueryId` in the model, (which is more descriptive / typesafe from the Java standpoint, as @dguy notes), while having it render to Json as just `"id" "CSAS_BLAH"` - just need to remember / find what the _thing_ was... I've done it before... hang on...

Reply:
Found it - you can add a `@JsonValue` to the `getId` or `toString` method of `QueryId` and it will render without the superfluous nesting and id field.... best of both worlds! :D

Reply:
@JsonValue rocks! :D

Reply:
Yep, how it is rendered via json etc is a different concern to how it is represented in the model.

Reply:
Agree on the separation of concerns. My point was that we should keep the api model classes decoupled from everything else. I'll add a new class for query ids in the model. I'll do this in another PR so that c3 is unblocked.